city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -14,8 +14,10 @@ class CookJusticeAdvisorySpider(CityScrapersSpider):
     agency = "Cook County Justice Advisory"
     timezone = "America/Chicago"
     allowed_domains = ["www.cookcountyil.gov"]
-    agenda_map = defaultdict(list)
-
+    
+    def __init__(self, agenda_map=None):
+        self.agenda_map = defaultdict(list)
```

Comment:
We'll need to call the superclass method here too like this https://github.com/City-Bureau/city-scrapers/blob/19fd52d76aa314734f620c4dafa9be0a0228ba81/city_scrapers/spiders/chi_teacherpension.py#L22-L24